store/src/sql/entities/event.rs

```diff
@@ -92,20 +92,11 @@ impl EventInsertable {
         self.deliverable = deliverable;
     }
 
-    /// Try to build the EventInsertable struct from a carfile.
-    pub async fn try_from_carfile(order_key: EventId, car_bytes: &[u8]) -> Result<Self> {
-        let event_cid = order_key.cid().ok_or_else(|| {
-            Error::new_invalid_arg(anyhow::anyhow!("EventID is missing a CID: {}", order_key))
-        })?;
-
-        if car_bytes.is_empty() {
-            return Err(Error::new_app(anyhow!(
-                "CAR file is empty: cid={}",
-                event_cid
-            )))?;
-        }
+    /// Underlying bytes that make up the event
+    pub async fn get_raw_blocks(&self) -> Result<Vec<EventBlockRaw>> {
+        let car = self.event.encode_car().await.map_err(Error::new_app)?;
```

Comment:
I hate this.  Encoding it to a CAR just to re-encode it to the raw blocks is doing two passes over the data for no reason.  I tried to add a method to unvalidated::Event that worked just like `encode_car()` but would instead return a `Vec<EventBlockRaw>` so we could get what we need in a single pass.  The problem is that the `EventBlockRaw` type lives in the `ceramic-store` crate while the `unvalidated::Event` type lives in the `ceramic-event` crate, but the store crate depends on the event crate, so I can't make the event crate depend on the store crate without introducing a circular dependency.

Would love any suggestions on how to clean this up.